ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/link/v1/link/link.html

```diff
@@ -0,0 +1,30 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Why could we not re-use the embed component having following configurations [1]

`{
  "type": "URL",
  "url": "https://www.pinterest.com/pin/146859637829777606/"
  },
  ":type": "core/wcm/components/embed/v1/embed"
}`


[1] https://aemcomponents.dev/content/core-components-examples/library/page-authoring/embed.html

Reply:
Inheriting from embed component is not helpful because we overwrite almost all logic
Rather aligning the html markup to embed element